apps/core/src/components/transaction/details/StakeTransactionDetails.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { IotaEvent } from '@iota/iota-sdk/client';
+import { formatPercentageDisplay } from '../../../utils';
+import { useGetValidatorsApy } from '../../../hooks';
+import { TransactionAmount } from '../amount';
+import { StakeTransactionInfo } from '../info';
+import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
+import type { GasSummaryType, RenderExplorerLink, RenderValidatorLogo } from '../../../types';
+
+interface StakeTransactionDetailsProps {
+    event: IotaEvent;
+    gasSummary?: GasSummaryType;
+    activeAddress: string | null;
+    renderExplorerLink: RenderExplorerLink;
+    renderValidatorLogo: RenderValidatorLogo;
```

Comment:
This prop `renderValidatorLogo` won't be needed at all after the refactor for #3916 since the ValidatorLogo will be moved to core